docker/Dockerfile

```diff
@@ -1,26 +1,29 @@
-FROM debian:stretch-20180426
+FROM debian:7.11-slim
 
 # TODO: This may be a moving target, figure out how to pin.
 RUN apt-get update \
-    && apt-get install -y \
+    && apt-get install -y --no-install-recommends \
         curl \
         gnupg2 \
         lsb-release \
         build-essential \
         procps \
         systemd \
+        ca-certificates \
+        adduser \
```

Comment:
For the future, should either of these two be a dependency of `google-fluentd`? Then they would be caught by `apt-get autoremove`.

Reply:
Created a ticket for this.